src/Workspaces/Core/Portable/Extensions/NotificationOptionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class NotificationOptionExtensions
+    {
+        public static string ToEditorConfigString(this NotificationOption notificationOption)
+        {
+            if (notificationOption == NotificationOption.Silent)
+            {
+                return nameof(NotificationOption.Silent).ToLowerInvariant();
+            }
+            else
+            {
+                return notificationOption.ToString().ToLowerInvariant();
```

Comment:
how is this different from the true clause?

Reply:
Without the if branch, we end up using "refactoring only" terminology as that is the underlying name of the [Silent notification option](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/CodeStyle/NotificationOption.cs,1cb65a33ddb41bc9). However, we want to use the term `silent` in editorconfig value, hence the special handling.

---
In reply to: [219242921](https://github.com/dotnet/roslyn/pull/30043#discussion_r219242921) [](ancestors = 219242921)